### ISSUE TITLE
virtualbox/eval-win10x86-enterprise times out during installing Guest Additions

### ISSUE THREAD

**Issue by spoorcc** [boxcutter/windows#125]
The similar x64 box builds correct for me, but when trying to create the x86 image it hangs on the `"e:\VBoxWindowsAdditions-x86.exe" /S` call in `vmtool.bat`. I tried logging in as mentioned in #119 but this did not resolve it. Guest additions is present in `E:\` and I can install it manually, but packer keeps waiting for the GuestAdditions command to return.

Below the logging with `@echo on` in `vmtool.bat`

```shell
==> virtualbox-iso: Provisioning with shell script: script/vmtool.bat
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>if not defined PACKER_SEARCH_PATHS set PACKER_SEARCH_PATHS="C:\Users\vagrant" a: b: c: d: e: f: g: h: i: j: k: l: m: n: o: p: q: r: s: t: u: v: w: x: y: z:
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>if not defined SEVENZIP_32_URL set SEVENZIP_32_URL=http://www.7-zip.org/a/7z1604.exe
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>if not defined SEVENZIP_64_URL set SEVENZIP_64_URL=http://www.7-zip.org/a/7z1604-x64.msi
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>if not defined VBOX_ISO_URL set VBOX_ISO_URL=http://download.virtualbox.org/virtualbox/5.1.22/VBoxGuestAdditions_5.1.22.iso
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>if not defined VMWARE_TOOLS_TAR_URL set VMWARE_TOOLS_TAR_URL=https://softwareupdate.vmware.com/cds/vmw-desktop/ws/12.5.5/5234757/windows/packages/tools-windows.tar
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>goto main
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>echo "virtualbox-iso"   | findstr /i "vmware"  1>nul
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>if not errorlevel 1 goto vmware
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>echo "virtualbox-iso"   | findstr /i "virtualbox"  1>nul
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>if not errorlevel 1 goto virtualbox
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>if exist "C:\Program Files (x86)" (set VBOX_SETUP_EXE=VBoxWindowsAdditions-amd64.exe )  else (set VBOX_SETUP_EXE=VBoxWindowsAdditions-x86.exe )
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>for %i in ("http://download.virtualbox.org/virtualbox/5.1.22/VBoxGuestAdditions_5.1.22.iso") do set VBOX_ISO=%~nxi
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>set VBOX_ISO=VBoxGuestAdditions_5.1.22.iso
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>set VBOX_ISO_DIR=C:\Users\vagrant\AppData\Local\Temp\virtualbox
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>set VBOX_ISO_PATH=C:\Users\vagrant\AppData\Local\Temp\virtualbox\VBoxGuestAdditions_5.1.22.iso
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>set VBOX_ISO=VBoxGuestAdditions.iso
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>mkdir "C:\Users\vagrant\AppData\Local\Temp\virtualbox"
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant>pushd "C:\Users\vagrant\AppData\Local\Temp\virtualbox"
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant\AppData\Local\Temp\virtualbox>set VBOX_SETUP_PATH=
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant\AppData\Local\Temp\virtualbox>if defined VBOX_SETUP_PATH goto install_vbox_guest_additions
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant\AppData\Local\Temp\virtualbox>if not exist a:\oracle-cert.cer echo ==> ERROR: File not found: a:\oracle-cert.cer   & goto exit1
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant\AppData\Local\Temp\virtualbox>echo ==> Installing Oracle certificate to keep install silent
    virtualbox-iso: ==> Installing Oracle certificate to keep install silent
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant\AppData\Local\Temp\virtualbox>certutil -addstore -f "TrustedPublisher" a:\oracle-cert.cer
    virtualbox-iso: TrustedPublisher "Trusted Publishers"
    virtualbox-iso: Certificate "Oracle Corporation" added to store.
    virtualbox-iso: CertUtil: -addstore command completed successfully.
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant\AppData\Local\Temp\virtualbox>echo ==> Installing VirtualBox Guest Additions
    virtualbox-iso: ==> Installing VirtualBox Guest Additions
    virtualbox-iso:
    virtualbox-iso: C:\Users\vagrant\AppData\Local\Temp\virtualbox>"e:\VBoxWindowsAdditions-x86.exe" /S
==> virtualbox-iso: Unregistering and deleting virtual machine...
==> virtualbox-iso: Error deleting virtual machine: VBoxManage error: VBoxManage.exe: error: Could not find a registered machine named 'eval-win10x86-enterprise-ssh'
==> virtualbox-iso: VBoxManage.exe: error: Details: code VBOX_E_OBJECT_NOT_FOUND (0x80bb0001), component VirtualBoxWrap, interface IVirtualBox, callee IUnknown
==> virtualbox-iso: VBoxManage.exe: error: Context: "FindMachine(Bstr(VMName).raw(), machine.asOutParam())" at line 152 of file VBoxManageMisc.cpp
==> virtualbox-iso: Deleting output directory...
Build 'virtualbox-iso' errored: Retryable error: Error removing temporary script at /tmp/script.bat: dial tcp 127.0.0.1:3544: connectex: No connection could be made because the target machine actively refused it.
```

**Comment by elatt** [comment]
I haven't run into this issue as my packer is using the guest additions .iso from the _host_ machine. However, there is a bug in that logic: it should not be using %VBOX_ISO_DIR% as the path to install the certs from.

**Comment by arizvisa** [comment]
Does this issue still occur? It seems that it's now using VBOX_SETUP_DIR to find the certificates... Should we bump the version from 5.1.30 to 5.2.44, or better yet 6.1.12? (I don't have vbox installed atm, so I can't test really).

(No worries if you don't care about this issue anymore. I'll mark it as stale and close it in a month if you've already moved past this and don't care to resolve it anymore).

**Comment by arizvisa** [comment]
Due to the lack of response on this issue, I've tagged it as stale and thus it will be closed in a month. If you feel this is in error, let me know and I'll untag.

**Comment by arizvisa** [comment]
Closing this issue as it has gone stale due to a lack of interaction. Despite this, this issue is suspected to have been fixed by PR #247. If you feel the closing of this is in error, please let me know and I will re-open.
